src/test/controllers/extension-limit-reached.spec.unit.ts

```diff
@@ -0,0 +1,80 @@
+import * as request from "supertest";
+import {ExtensionsCompanyProfile, getCompanyProfile} from "../../client/apiclient";
+import * as mockUtils from "../mock.utils";
+import app from "../../app";
+import * as pageURLs from "../../model/page.urls";
+import {COOKIE_NAME} from "../../session/config";
+import { loadSession } from "../../services/redis.service";
+import {createHistoryIfNone, getCompanyInContext} from "../../services/session.service";
+
+jest.mock("../../client/apiclient");
+jest.mock("../../services/redis.service");
+jest.mock("../../services/session.service");
+
+const mockCompanyProfile: jest.Mock = (<unknown>getCompanyProfile as jest.Mock<typeof getCompanyProfile>);
```

Comment:
you should be able to replace the (<unknown..... as jest.Mock<typeOf..) with just as jest.Mock
eg. const mockCompanyProfile: getCompanyProfile as jest.Mock

same for the others

Reply:
Changed this.